posts/2020-12-15-openliberty-realworld-example-app.adoc

```diff
@@ -14,15 +14,15 @@ David Shi <https://github.com/dshimo>
 
-link:https://github.com/gothinkster/realworld[RealWorld] serves as your fitting room for building "real world" applications. Created as an open source project, RealWorld enables users to choose any combination of frontend and backend to see how they work together in a sample app called link:https://demo.realworld.io/["Conduit"]. 
+Let link:https://github.com/gothinkster/realworld[RealWorld] serve as your fitting room for playing around and building "real world" applications. Created as an open source project, RealWorld enables users to choose any combination of frontend and backend to see how they work together in a sample app called link:https://demo.realworld.io/["Conduit"]. This provides developers a great sandbox environment for exploring new or familiar technologies. What if you're a frontend developer with no interest in backend, or vice versa? Since the frontend or backend can run independently of each other, you can for example, dedicate your efforts to finding the frontend that suits you without knowing any details of how the backend performs its tasks. RealWorld manages this by enforcing link:https://github.com/gothinkster/realworld/tree/master/spec#project-overview[API specs] for Conduit's endpoints.
 
-For today's blog post, I'm taking advantage of several link:https://www.openliberty.io/guides/[Open Liberty guides] to build the link:https://github.com/OpenLiberty/openliberty-realworld-example-app[backend server] for RealWorld by using Open Liberty and MicroProfile. 
+For today's blog post, I'll be exploring some MicroProfile technologies and link:https://www.openliberty.io/guides/[Open Liberty guides] for powering Conduit's backend on Open Liberty. Specifically, we'll be propping up a link:https://en.wikipedia.org/wiki/Jakarta_RESTful_Web_Services[JAX-RS] application for our REST endpoints, adding some security to them, and then persisting our data with link:https://en.wikipedia.org/wiki/Jakarta_Persistence[JPA].
 
 == Starting a RESTful App
 
-Let's begin by deploying a JAX-RS application with some endpoints so we can start playing with it. By following link:https://openliberty.io/guides/rest-intro.html[guide to create a RESTful web service], we can prop up an endpoint to return simple messages or JSON-formatted classes. As a simplified example, for our blogging website we can start with an endpoint to return an instantiated user:
+Let's begin by deploying a JAX-RS application with some initial endpoints so we can start playing with it. By following link:https://openliberty.io/guides/rest-intro.html[guide to create a RESTful web service], we can prop up an endpoint to return simple messages or JSON-formatted classes. As a simplified example, for our blogging website we can start with an endpoint to return an instantiated user:
```

Comment:
By following link:https://openliberty.io/guides/rest-intro.html[guide to create a RESTful web service]
--->
By following the link:https://openliberty.io/guides/rest-intro.html[guide to create a RESTful web service]

for our blogging website
what is the blogging website? It isn't mentioned elsewhere or explained- is it related to the conduit app?

Reply:
yes Conduit is the blogging website.
i'll replaced that term with Conduit